### PR TITLE
Split rgw suites to reduce execution time

### PR DESCRIPTION
**Pull request by ckulal**
Signed-off-by: ckulal <ckulal@redhat.com>

splitting rgw pacific ms related test-suites to reduce execution time

tier-2_rgw_ssl_ecpool_ms-bucket-objects-gc-policy-on-secondary.yaml
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-FO5ZJD/

tier-2_rgw_ssl_ecpool-ms-bucket-listing-versioning-on-secondary.yaml
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-BE6PH1/

tier-1_rgw_ms-omap-datalog-on-primary.yaml
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-0KE94Z/

tier-1_rgw_ms-bilog-crash-on-primary.yaml
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-GHI4OV/

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: pipeline/metadata/5.1.yaml [hunk 1]
```diff
@@ -414,8 +429,23 @@
...
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - sanity
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
```
Comment (rkhambad) [inline pipeline/metadata/5.1.yaml:446]: you can add stage-4 for tier-2  in 5.1.yaml  , as we are maintaining only 5 test suite  in a single stage.

File: pipeline/metadata/5.2.yaml [hunk 2]
```diff
@@ -492,8 +492,24 @@
...
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - sanity
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
```
Comment (rkhambad) [inline pipeline/metadata/5.2.yaml:509]: here also we are already having 5 test suite in stage-3 and stage-5, you can refer link https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=rhcsqe&title=Test+Suites 
and can adjust the stage or add new stage (like stage-6 for tier-2) for newly extra added test suite.

Comment (ckulal) [reply pipeline/metadata/5.2.yaml:509]: done

**Review by mkasturi18 — COMMENTED** [review]
The changes looks good to me.

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**